front/app/utils/moduleUtils.tsx

```diff
@@ -365,19 +366,19 @@ export interface OutletsPropertyMap {
     onMount: () => void;
   };
   'app.containers.Admin.settings.customize.headerSectionStart': {
-    homepageSettings: IHomepageSettingsAttributes;
+    bannerLayout: THomepageBannerLayout;
     handleOnChange: (
-      settingKey: keyof IHomepageSettingsAttributes,
-      settingValue: any
+      settingKey: 'banner_layout',
+      settingValue: THomepageBannerLayout
     ) => void;
   };
   'app.containers.Admin.settings.customize.headerSectionEnd': {
-    homepageSettings: IHomepageSettingsAttributes;
+    homepageSettings: Partial<IHomepageSettingsAttributes>
```

Comment:
The types for this one are still broken, as it expects full `IHomepageSettingsAttributes` (we were passing in the full object before) but now it should just be a subset of that